docs/creating_component_code_review.md

```diff
@@ -47,8 +49,15 @@ bridge = Bridge(...)
 status = bridge.status()
 ```
 
-### 5. Limit platforms included in initial pull request
-Large pull requests mean there is a larger chance of finding problems that need to be addressed, and more code that needs to be reviewed between every requested change. If your new component will have multiple platforms, try and limit your initial pull request to a single platform. Once the initial component is merged, you can submit additional PRs for the remaining platforms. This allows reviewers to sign off on smaller chunks of code one at a time, and lets us get your new feature in sooner. Pull requests containing large code dumps will not be a priority for review.
+### 5. Limit initial pull request to Minimum Viable Product (MVP)
+Large pull requests mean there is a larger chance of finding problems that need to be addressed, and more code that needs to be reviewed between every requested change. Limit the initial pull request to the minimum functionality needed for someone to get value out of the integration. Once the initial pull request is merged, you can submit additional PRs for the remaining features. This allows reviewers to sign off on smaller chunks of code one at a time, and lets us get your new integration/features in sooner. **Pull requests containing large code dumps (> 800 line adds excluding tests) will not be a priority for review and may be closed.** Limit MVPs to the following:
+- A single platform
+- No custom services 
+- Loaded via `configuration.yaml` _or_ config flow (if `configuration.yaml` cannot be used)
```

Comment:
This should be rephrased; there should be no priority between the two and configuration.yaml is not a requirement. Especially since config entries enable a lot of other neat features down the line of the components life.